clusters/build-clusters/01_cluster/openshift/ci-operator/regcred_secret_template.yaml

```diff
@@ -11,7 +11,7 @@ objects:
         * by the image clients in ci-operator pod: the local images synced by dptp-controller are eventually consistent only
         * by the build pods created by ci-operator: https://bugzilla.redhat.com/show_bug.cgi?id=1861546
 
-    name: regcred
+    name: registry-pull-credentials
```

Comment:
Please do not change the existing stuff, just add the new secret and keep the old one the way they are. This is sensitive and can result in huge outages. Let's do this properly in well isolated steps.